File: monorepo_java/apps/web/src/main/java/mono/web/CycleB.java
package mono.web;

public class CycleB {
    private final String tag;
    private CycleA owner;                        // C1.b.type_ref → CycleA

    public CycleB(String tag) { this.tag = tag; }

    public String bounceToAlpha() {
        return new CycleA("bounce-from-" + tag).describe();
    }
}

File: monorepo_php/apps/web/src/CycleB.php
<?php
namespace Mono\Web;

class CycleB {
    public ?CycleA $owner = null;                    // C1.b.type_ref → CycleA

    public function __construct(public readonly string $tag) {}

    public function bounceToAlpha(): string {
        return (new CycleA("bounce-from-{$this->tag}"))->describe();
    }
}
